lib/widgets/message_list.dart

```diff
@@ -77,13 +98,15 @@ class MessageListAppBarTitle extends StatelessWidget {
 
       case StreamNarrow(:var streamId):
         final store = PerAccountStoreWidget.of(context);
-        final streamName = store.streams[streamId]?.name ?? '(unknown stream)';
-        return Text("#$streamName"); // TODO show stream privacy icon
+        final stream = store.streams[streamId];
+        final streamName = stream?.name ?? '(unknown stream)';
+        return _buildStreamRow(stream, streamName);
 
       case TopicNarrow(:var streamId, :var topic):
         final store = PerAccountStoreWidget.of(context);
-        final streamName = store.streams[streamId]?.name ?? '(unknown stream)';
-        return Text("#$streamName > $topic"); // TODO show stream privacy icon; format on two lines
+        final stream = store.streams[streamId];
+        final streamName = stream?.name ?? '(unknown stream)';
+        return _buildStreamRow(stream, "$streamName > $topic");
```

Comment:
```
msglist: Show stream privacy icon in app bar
```

As discussed in the office just now, for stream and topic narrows, I notice on iOS that title part of the app bar is now start-aligned rather than centered; it seems we've lost the default behavior that should come from omitting [`centerTitle`](https://api.flutter.dev/flutter/material/AppBar/centerTitle.html). It probably makes sense to keep it centered on iOS.

Reply:
Hmm yeah, thanks for catching that. Will fix.